azavea/react-showtime
# Deploy example site to Netlify

## Overview

Creates a Netlify config file to build and deploy the `example/` subdirectory to https://develop--react-showtime.netlify.app.

Resolves #1 

### Notes

This repository was made public to allow Netlify to install `react-showtime` from this repository without needing to set up basic auth with access tokens embedded in `package.json`.

## Testing Instructions

- Verify that the [Netlify build](https://app.netlify.com/sites/react-showtime/deploys/5fb53d0616c81100085d7740) passes.
- Verify that the [deploy preview](https://deploy-preview-6--react-showtime.netlify.app/) is available with no content security policy errors in the console.
- Run `yarn install` in the repository root. Run `yarn link` to use the local version of the package when installing on the example site.
- `cd` into the `example` subdirectory.
- Run `yarn link react-showtime` to link the local package in the parent directory. Run `yarn install`.
- Run `yarn start`. Confirm that the site runs at http://localhost:3000/ without errors.
  - There will be a brief error message in the terminal about not being able to load `eslint-loader`. This is an informational message and [recognized issue](https://github.com/gsoft-inc/craco/issues/205) with CRA v4, and doesn't actually impact Yarn's ability to run `eslint`.
